mappings/net/minecraft/world/gen/feature/TreeConfiguredFeatures.mapping

```diff
@@ -7,6 +8,7 @@ CLASS net/minecraft/unmapped/C_fxqldlqj net/minecraft/world/gen/feature/TreeConf
 	FIELD f_dgflpfie SPRUCE Lnet/minecraft/unmapped/C_xhhleach;
 	FIELD f_dsunpqgn MANGROVE Lnet/minecraft/unmapped/C_xhhleach;
 	FIELD f_eyldriwt ACACIA Lnet/minecraft/unmapped/C_xhhleach;
+	FIELD f_fboflfct CHERRY_BEES_005 Lnet/minecraft/unmapped/C_xhhleach;
 	FIELD f_gnvxdkri BIRCH_BEES_005 Lnet/minecraft/unmapped/C_xhhleach;
```

Comment:
completely unrelated but remember when this field was `BICH_BEES_005` for like two years